app/models/journeys.rb

```diff
@@ -0,0 +1,48 @@
+# Policies should not control the front-end journey as one journey can be an application
+# for more than one policy
+class Journeys
```

Comment:
Prefer singular naming.
```suggestion
class Journey
```